pkg/registry/core/event/strategy.go

```diff
@@ -84,9 +84,10 @@ func GetAttrs(obj runtime.Object) (labels.Set, fields.Set, error) {
 	if !ok {
 		return nil, nil, fmt.Errorf("not an event")
 	}
-	return labels.Set(event.Labels), EventToSelectableFields(event), nil
+	return labels.Set(event.Labels), ToSelectableFields(event), nil
 }
 
+// MatchEvent returns events matching a selection predicate
```

Comment:
it returns a predicate not events

Reply:
BTW - it would be good to unify naming across different package, I think there are two schemes currently:
- Matcher
- Match[type-name]
[I personally thing Matcher is better because it's the same everywhere, but at least being consistent with it would be good]

Reply:
I renamed the function to `Matcher`.